PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -3,6 +3,9 @@
 from PhysicsTools.SelectorUtils.tools.vid_id_tools import *
 
 def miniAOD_customizeCommon(process):
+    #slimmed pileup information
+    process.load('PhysicsTools.PatAlgos.slimming.slimmedAddPileupInfo_cfi')
```

Comment:
the way this module is currently implemented, it should be only in miniAOD_customizeMC
PileupSummaryInfo itself would not fit the info available in real data